tests/manage/monitoring/conftest.py

```diff
@@ -143,3 +143,75 @@ def stop_mgr():
     logger.info(f"Upscaling deployment {mgr} back to 1")
     oc.exec_oc_cmd(f"scale --replicas=1 deployment/{mgr}")
     return measured_op
+
+
+@pytest.fixture(scope="session")
+def workload_stop_ceph_mon():
+    """
+    Downscales Ceph Monitor deployment, measures the time when it was
+    downscaled and monitors alerts that were triggered during this event.
+
+    Returns:
+        dict: Contains information about `start` and `stop` time for stopping
+            Ceph Monitor pod
+    """
+    oc = ocp.OCP(
+        kind=constants.DEPLOYMENT,
+        namespace=config.ENV_DATA['cluster_namespace']
+    )
+    mon_deployments = oc.get(selector=constants.MON_APP_LABEL)['items']
+    mons = [
+        deployment['metadata']['name']
+        for deployment in mon_deployments
+    ]
+
+    # get monitor deployments to stop, leave even number of monitors
+    split_index = len(mons) // 2 if len(mons) > 3 else 2
+    mons_to_stop = mons[split_index:]
+    logger.info(f"Monitors to stop: {mons_to_stop}")
+    logger.info(f"Monitors left to run: {mons[:split_index]}")
+
+    def stop_mon():
+        """
+        Downscale Ceph Monitor deployments for 12 minutes. First 15 minutes
+        the alert CephMonQuorumAtRisk should be in 'Pending'. After 15 minutes
+        the alert turns into 'Firing' state.
+        This configuration of monitoring can be observed in ceph-mixins which
+        are used in the project:
+            https://github.com/ceph/ceph-mixins/blob/d22afe8c0da34490cb77e52a202eefcf4f62a869/config.libsonnet#L16
```

Comment:
is this configuration adjustable? if yes, maybe it would be better to fetch the value of monQuorumAlertTime during the test instead of hardcoding 15 minutes in the test?

Reply:
I believe that we shouldn't take the value directly from this ceph-mixins repo as it is upstream. If it is changed then there is a possibility that we wouldn't notice. If we hardcode it then we can see that the value changed because the test would fail.

Reply:
ack